tests.py

```diff
@@ -147,126 +169,126 @@ def run_vsm(self, name, input_data, expected_output,
     def test_simple0(self):
         input_data = 'transmission_gear = "reverse"'
         expected_output = '''
-transmission_gear,[SIGNUM],'reverse'
+transmission_gear,1,'reverse'
 State = {
 transmission_gear = reverse
 }
-car.backup,[SIGNUM],'True'
+car.backup,3,'True'
 State = {
 car.backup = True
 transmission_gear = reverse
 }
 condition: (transmission_gear == 'reverse') => True
-transmission_gear,[SIGNUM],'"reverse"'
-car.backup,[SIGNUM],'True'
+transmission_gear,1,'"reverse"'
+car.backup,3,'True'
         '''
         self.run_vsm('simple0', input_data, expected_output.strip() + '\n')
 
     def test_simple0_delayed(self):
         input_data = 'transmission_gear = "reverse"'
         expected_output = '''
-transmission_gear,[SIGNUM],'reverse'
+transmission_gear,1,'reverse'
 State = {
 transmission_gear = reverse
 }
 condition: (transmission_gear == 'reverse') => True
-car.backup,[SIGNUM],'True'
+car.backup,3,'True'
 State = {
 car.backup = True
 transmission_gear = reverse
 }
-transmission_gear,[SIGNUM],'"reverse"'
-car.backup,[SIGNUM],'True'
+transmission_gear,1,'"reverse"'
+car.backup,3,'True'
         '''
         self.run_vsm('simple0_delay', input_data, expected_output.strip() + '\n')
 
     def test_simple0_uninteresting(self):
         input_data = 'phone_call = "inactive"'
         expected_output = '''
-phone_call,[SIGNUM],'inactive'
+phone_call,2,'inactive'
 State = {
 phone_call = inactive
 }
 condition: (phone_call == 'active') => False
-phone_call,[SIGNUM],'"inactive"'
+phone_call,2,'"inactive"'
         '''
         self.run_vsm('simple0', input_data, expected_output.strip() + '\n',
                 send_quit=True)
 
     def test_simple2_initial(self):
         input_data = 'damage = true'
         expected_output = '''
-damage,[SIGNUM],True
+damage,5,True
 State = {
 damage = True
 moving = false
 }
-car.stop,[SIGNUM],'True'
+car.stop,4,'True'
 State = {
 car.stop = True
 damage = True
 moving = false
 }
 condition: (moving != True and damage == True) => True
-damage,[SIGNUM],'true'
-car.stop,[SIGNUM],'True'
+damage,5,'true'
+car.stop,4,'True'
```

Comment:
It's a bit weird that some are true and some are True, I guess I know why that is, but maybe we should add a change to normalise that?